templates/email/oxfordshire/alert-update.html

```diff
@@ -0,0 +1,34 @@
+[%
+
+title = report.title | html;
+email_summary = "New updates on “" _ title _ "”";
+email_columns = 2;
+
+PROCESS '_email_settings.html';
+
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+  <h1 style="[% h1_style %]">NEW updates on <a href="[% problem_url %]">[% title %]</a></h1>
+  [%~ INCLUDE '_email_comment_list.html' %]
+  [% IF problem.is_closed ~%]
+    <a href="https://youtu.be/hae9g170sA4">
+      <p style="[% p_style %]">Cllr Tim Bearder explains when we don’t fix things</p>
+      <img style="width: 300px" alt="Understanding when we don’t fix things video" src="/cobrands/oxfordshire/images/oxfordshire-can-cants-video.png">
+    </a>
+  [%~ END %]
+ [% IF unsubscribe_url %]
+  <p style="[% p_style %]"><a href="[% unsubscribe_url %]">Unsubscribe from alerts about this report</a></p>
+ [% END %]
+  [% end_padded_box | safe %]
+</th>
+[% WRAPPER '_email_sidebar.html' object = report %]
+    <h2 style="[% h2_style %]">[% title %]</h2>
+    <p style="[% secondary_p_style %]">[% report.detail | html %]</p>
+    [% TRY %][% INCLUDE '_council_reference_alert_update.html' problem=report p_style=secondary_p_style %][% CATCH file %][% END %]
```

Comment:
Do we want the video at the bottom of the left column, or could it go the bottom of the right column? If it can go bottom of the right, you don't need to override alert-update.html/txt, you can put the video in this template and then it will be included there.

Reply:
@dracos This was my reasoning to place the video at the bottom of the left column:

I didn't place the video under the report details, It wouldn't be very visible, especially on mobile, where is going to sit right at the bottom of the email. I'm guessing the council really want the user to watch the video, so they don't call complaining straight away. So I placed the video right under the "Closed" status, so the user will see that the issue wasn't fixed, but they will have something explaining why not straight away.

Reply:
Okay, though there might be more than one update conceivably :) (but probably not often.) That's fine, but what we should do is add a new INCLUDE template at the place you want to put the video so that you don't need to override the whole alert-update template, but instead can add an oxfordshire specific template of just the needed bit, similar to how the include is done here.

`_alert_update_after_updates.html` would be the first name I think of.

Reply:
Awesome I just push a commit for with this change.